webant/webant.py

```diff
@@ -18,7 +18,52 @@
 from webserver_utils import gevent_run
 
 
-def create_app(configfile=None):
+class LibreantCoreApp(Flask):
+    def __init__(self, import_name):
+        super(LibreantCoreApp, self).__init__(import_name)
+        self.config.update({
+            'BOOTSTRAP_SERVE_LOCAL': True,
+            'DEBUG': True,
+            'PRESET_PATHS': [],  # defaultPreset should be loaded as default?
+            'FSDB_PATH': "",
+            'AGHERANT_DESCRIPTIONS': [],
```

Comment:
I don't like that ```AGHERANT_DESCRIPTIONS```, ```PRESET_PATHS``` and other child-class related configuration are initialized in LibreantCoreApp.
We can add a dict parameter to the ```__init__``` function in order to pass defaults from sub-classes.